indra/db/database_manager.py

```diff
@@ -324,6 +324,7 @@ class PreassemblyUpdates(self.Base):
         class PAStatements(self.Base):
             __tablename__ = 'pa_statements'
             mk_hash = Column(BigInteger, primary_key=True)
+            matches_key = Column(String, unique=True, nullable=False)
```

Comment:
This should detect if there is a hash collision (very unlikely, but good to have a check)